app/views/va_facilities/_visn_combo_box.html.erb

```diff
@@ -0,0 +1,15 @@
+<label for="<%= visn_selector %>" class="usa-label line-height-26">VISN</label>
+<div class="margin-top-1 margin-bottom-2 desktop:margin-y-0">
```

Comment:
I don't think we need to specify `desktop` here since these pages are not mobile-friendly.